python/gps/algorithm/traj_opt/traj_opt_pi2.py

```diff
@@ -0,0 +1,163 @@
+""" This file defines code for PI2-based trajectory optimization.
+
+Optimization of trajectories with PI2 and a REPS-like KL-divergence constraint. 
+References:
+[1] E. Theodorou, J. Buchli, and S. Schaal. A generalized path integral control 
```

Comment:
Maybe also add a reference to your PIGPS paper at the top of PIGPS algorithm file?